test/pamela/HTN/parallel-constraints.main.tpn.edn

```diff
@@ -1,168 +1,264 @@
 {:act-104
- {:args ["host0301" "host0501" "1"],
-  :argsmap {"client" "host0301", "display" "1", "server" "host0501"},
+ {:args
```

Comment:
When we construct the HTNs and the TPNs, we used to (and should still) dereference things like `loc2` into their values (e.g., `"host0301"`).